handbook/engineering/campaigns/index.md

```diff
@@ -67,6 +67,21 @@ When I deploy a static site on Netlify I want to focus on my site — its conten
 
 When I create a campaign to make large-scale code changes I want to _focus on the specific change I want to make across all of the code at my organization_. I don't want to worry about all of the overhead associated with execution, code hosts, and management of all things listed above.
 
+## Process
+
+For each iteration (currently one month long), we follow this process:
+
+* Before the iteration begins, each team member prepares for the iteration planning meeting by considering what they would like to see in the next iteration, and by looking over [our backlog](https://github.com/sourcegraph/sourcegraph/labels/team%2Fcampaigns) and setting the Milestone for issues we'd like to include.
+* We then have our planning meeting to determine our common goals for the iteration. We create a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address.
+* Each Monday, the team meets with stakeholders to stay in sync, and to set our personal goals for the week. We record these goals, and the previous week's progress, as status update comments on the tracking issue. The engineering manager then compiles these into a summary which is sent to the team and stakeholders.
+* Each day, Slack reminds us to do our stand-up, which consists of a *short* message in the reminder's thread. As we are a globally distributed team, this can be a statement of intent for the day, or a recap of what we have finished that day.
```

Comment:
I really liked how you phrased that when proposing it, since it makes the expectations _really_ clear, so maybe we can include that?

```suggestion
* Each day, Slack reminds us to do our stand-up, which consists of a *short* message (it shouldn't take longer than 30s to write) in the reminder's thread. As we are a globally distributed team, this can be a statement of intent for the day, or a recap of what we have finished that day.
```